.gitignore

```
@@ -3,4 +3,5 @@
 dist
 shipstation.egg-info
 .tox
-.idea
\ No newline at end of file
+.idea
+build/lib/*
```

Looks like this didn't have a newline at the end of the file, should probably add that in now. 
```suggestion
build/lib/*

```